contrib/google.golang.org/internal/grpcutil/tag_utils.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package grpcutil // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/google.golang.org/internal/grpcutil"
+import (
+	"strings"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+)
+
+// ExtractRPCTags will assign the proper tag values for method, service, package according to otel given a full method
+func ExtractRPCTags(fullMethod string) map[string]string {
+
+	//Otel definition: https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/rpc/#span-name
+
+	tags := map[string]string{
+		ext.RPCMethod:   "",
+		ext.RPCService:  "",
+		ext.GRPCPackage: "",
+	}
+
+	//Always remove leading slash
+	fullMethod = strings.TrimPrefix(fullMethod, "/")
+
+	//Split by slash and get everything after last slash as method
+	slashSplit := strings.SplitAfter(fullMethod, "/")
+	tags[ext.RPCMethod] = slashSplit[len(slashSplit)-1]
```

Comment:
You should check the length of the result after splitting to make sure it doesn't break when receiving unexpected strings.

```suggestion
	elems := strings.Split(strings.TrimPrefix(fullMethod, "/"), "/")
	if len(elems) != 2 {
	   rpcMethod = "unknown"
	   rpcService = "unknown"
	   ...
	   return
	}
	
    rpcService, rpcMethod = elems[0], elems[1]	
```

Reply:
Are we sure that there are only two elements after splitting by '/'? Again, refer to comments down below on the Otel definition, but if there are multiple slashes in the package, then this would not work so for now ignoring this